features/support/twopence_init.rb

```diff
@@ -41,22 +47,17 @@
 
 # This function is used to get one of the nodes based on its type
 def get_target(host)
-  case host
-  when 'server'
-    node = $server
-  when 'ceos-minion'
-    node = $ceos_minion
-  when 'ssh-minion'
-    node = $ssh_minion
-  when 'sle-minion'
-    node = $minion
-  when 'sle-client'
-    node = $client
-  when 'sle-migrated-minion'
-    node = $client
-  else
-    raise 'Invalid target.'
-  end
+  map = {
+    'server' => $server,
+    'proxy' => $proxy,
+    'ceos-minion' => $ceos_minion,
+    'ssh-minion' => $ssh_minion,
+    'sle-minion' => $minion,
```

Comment:
2. the newer syntax in ruby is more readable

```ruby
 { key: 'value' }
'sle-minion': $minion,
```

Reply:
Right.

Reply:
create this like that 

```ruby
options = { font_size: 10, font_family: "Arial" }
````

Reply:
Not sure I understand.
```
sle-minion: $minion
```
? Please notice "sle-minion" is a string, not a key.

Reply:
why do you need a string: 

```ruby

#! /usr/bin/ruby

def get_target(host)
  $server = 'baubau'
  $proxy = 'mia'
  nodes_map = {
    server:  $server,
    proxy:  $proxy
  }
  node = nodes_map[host.to_sym]
  puts node
  node
end

grande_host = 'server'

get_target(grande_host)
```

Reply:
No. `$server` contains a node, not a string. I cannot do $server = 'baubau'.

Sorry, i stop losing time with this -- `=>` is just fine.

Please come up with a working and tested solution and prepare a pull request if youfind the way to do it. But I won't spend more time on something's that just cosmetic.

Reply:
the solution i provided it works. For ruby having a Twopence Node or Baubau Object doesn't matter.
(Baubau was an example sorry!! ) 
At this point i will refactor it later, if you cannot and are fine

Reply:
thanks for your undestanding
:+1: